infra/base-images/base-runner/Dockerfile

```diff
@@ -37,19 +37,46 @@ COPY --from=base-clang /usr/local/bin/llvm-cov \
 
 RUN apt-get update && apt-get install -y \
     binutils \
+    ca-certificates \
     file \
     fonts-dejavu \
     git \
     lib32gcc1 \
     libc6-i386 \
     libcap2 \
-    python3 \
-    python3-pip \
-    python3-setuptools \
     unzip \
     wget \
     zip --no-install-recommends
 
+# Build and install latest Python 3 (3.8.3).
+ENV PYTHON_VERSION 3.8.3
+RUN export PYTHON_DEPS="\
```

Comment:
I think we might as well do this in base-image at this point and save some build time, maintenance burden, and